protocol/v2/ssv/runner/committee.go

```diff
@@ -36,30 +37,31 @@ type CommitteeRunner struct {
 	BaseRunner     *BaseRunner
 	network        specqbft.Network
 	beacon         beacon.BeaconNode
-	signer         spectypes.BeaconSigner
-	operatorSigner ssvtypes.OperatorSigner
+	signer         types.BeaconSigner
+	operatorSigner types.OperatorSigner
+	domain         spectypes.DomainType
 	valCheck       specqbft.ProposedValueCheckF
 
 	stoppedValidators map[spectypes.ValidatorPK]struct{}
-	submittedDuties   map[spectypes.BeaconRole]map[phase0.ValidatorIndex]struct{}
+	submittedDuties   map[types.BeaconRole]map[phase0.ValidatorIndex]struct{}
 
 	started time.Time
 	metrics metrics.ConsensusMetrics
 }
 
 func NewCommitteeRunner(
 	networkConfig networkconfig.NetworkConfig,
-	share map[phase0.ValidatorIndex]*spectypes.Share,
+	share map[phase0.ValidatorIndex]*types.Share,
```

Comment:
be consistent with all the project, importing the types from spec should use the name `spectypes` and not `types`